### PR TITLE
Spanish Latam Language

### PR DESCRIPTION
**Pull request by Matyrela**
**Describe your change**

Added the Spanish Latam because I did not agree with some translations
I do not change the current Spanish translation.

**Translation checklist**

- [x] Have you run the application to make sure your code runs
- [x] Have you used the correct keys for your translations added (e.g. `en`) 
- [x] Have you made sure to leave a `,` at the end of each translation added
- [x] Have you added the language to `supportedLanguages` in alphabetical order
- [x] Have you added the language to the table in README.md in alphabetical order
- [ ] If you have translated the README, have you linked it in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,8 +105,8 @@ If you're having issues with the packaged executable or using this tool in gener
 | ------------------------------------------- | ----- | --------------------------------------------- | ------------------------------------ |
 | Arabic (العربية)                            | ar    | [Tayeb-Ali](https://github.com/tayeb-ali)     | UI                                   |
 | Brazilian Portuguese (Português Brasileiro) | pt_br | [marleyas](https://github.com/marleyas)       | UI (partial)                         |
-| Chinese Simplified (简体中文)               | zh    | [jiangzhe11](https://github.com/jiangzhe11)   | UI and [README](./README-Chinese.md) |
-| Chinese Traditional (繁體中文)              | zh_tw | [startgo](https://github.com/ystartgo)        | UI                                   |
+| Chinese Simplified  (简体中文)               | zh    | [jiangzhe11](https://github.com/jiangzhe11)   | UI and [README](./README-Chinese.md) |
+| Chinese Traditional  (繁體中文)              | zh_tw | [startgo](https://github.com/ystartgo)        | UI                                   |
```
Comment (brentvollebregt) [inline README.md:109]: Has this been done using an auto-formatter? These changes don't show up when I auto format using prettier in vscode

File: auto_py_to_exe/web/index.html [hunk 2]
```diff
@@ -1,246 +1,263 @@
 <!DOCTYPE html>
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/index.html:1]: What are the changes in this file for? There are a lot of unnecessary changes.

File: auto_py_to_exe/web/js/i18n.js [hunk 3]
```diff
@@ -18,6 +19,24 @@ const translationMap = {
...
+                sp_la: 'Idioma ',
+                tr: 'Deyim ',
+                th: 'สำนวน ',
+                fr: 'Langage ',
+                it: 'Idioma ',
+                id: 'Idiom ',
+                cz: 'Idiom ',
+                gr: 'ιδίωμα ',
+                ar: 'لغة. مثل',
+            },
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:39]: Are these related to a change in index.html?

**Review by brentvollebregt — CHANGES_REQUESTED** [review]

File: auto_py_to_exe/web/js/i18n.js (hunk 3 above)
Comment (Matyrela) [reply auto_py_to_exe/web/js/i18n.js:39]: Yes, because when you change the language the text "Language"  in the UI doesnt change to the one related

Comment (brentvollebregt) [reply auto_py_to_exe/web/js/i18n.js:39]: I see - any need for the spaces at the end of each of these strings?

Comment (Matyrela) [reply auto_py_to_exe/web/js/i18n.js:39]: For a little of space between the language drop down list, nothing important

File: README.md (hunk 1 above)
Comment (Matyrela) [reply README.md:109]: It was auto-formatter, nothing important

File: auto_py_to_exe/web/js/i18n.js (hunk 3 above)
Comment (brentvollebregt) [reply auto_py_to_exe/web/js/i18n.js:39]: Before we had "Language:" but now the colon (`:`) isn't showing - could we get that back? Preferably keeping the colon out of the translation file and in the index.html (after the `small` tag with the translation on it.

Also removing the spaces from here would be fine - if spacing is needed in the UI (there should be natural spacing due to the spacing between tags), you can add a 4px left margin to the dropdown in CSS.

If you're not sure about any of this or want me to help - let me know 👌

Comment (Matyrela) [reply auto_py_to_exe/web/js/i18n.js]: Ok, Everything done!

File: auto_py_to_exe/web/index.html [hunk 4]
```diff
@@ -243,4 +245,4 @@ <h2 data-i18n="ui.title.output">Output</h2>
             </div>
         </div>
     </body>
-</html>
+</html>
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/index.html:248]: Can you revert this change? I don't believe it's needed.

**Review by brentvollebregt — COMMENTED** [review]
Tested it locally and it's looking good. Just one minor thing I saw in the diff - after that I can merge and create a release.

**Review by brentvollebregt — APPROVED** [review]

**Comment by brentvollebregt** [comment]
Thank you very much! This is going out in v2.25.0 right now - should be ready in 10mins
